test/AutoRest.TestServer.Tests/lro.cs

```diff
@@ -480,27 +472,19 @@ public class LroTest: TestServerTestBase
         });
 
         [Test]
-        [Ignore("Handle multiple responses: https://github.com/Azure/autorest.csharp/issues/413")]
         public Task LROErrorPut201NoProvisioningStatePayload() => Test(async (host, pipeline) =>
         {
             var value = new Product();
             var operation = await new LrosaDsClient(ClientDiagnostics, pipeline, host).StartPutError201NoProvisioningStatePayloadAsync(value);
-            var result = await operation.WaitForCompletionAsync().ConfigureAwait(false);
-            Assert.AreEqual("100", result.Value.Id);
-            Assert.AreEqual("foo", result.Value.Name);
-            Assert.AreEqual("Succeeded", result.Value.ProvisioningState);
+            Assert.ThrowsAsync<RequestFailedException>(async () => await operation.WaitForCompletionAsync().ConfigureAwait(false));
```

Comment:
Not sure if this one should throw. Can you double check if it does in python?

Reply:
`LROError` are error situations. Here, we go down a failure path, which throws.

Reply:
Then we need to fix LROErrorDeleteNoLocation_Sync

Reply:
So, `LROErrorDeleteNoLocation` simply does:
```
res.status(204).end();
```
Which means it is a DELETE operation that returns 204 (which is valid). It succeeds because of:
```csharp
// If provisioningState was not found, it defaults to Succeeded.
if (_headerFrom == HeaderFrom.None)
{
    return true;
}
```
This is supposed to be how you handle a terminal polling state in this situation. So, I don't know what is an 'error' about this test. I'll check python to see what they are doing.